doc/nrf/releases/release-notes-2.4.1.rst

```diff
@@ -0,0 +1,142 @@
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.4.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.4.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_241_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+See `Matter samples`_ for the list of changes for the Matter samples.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.1.0.1`` tag.
+
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added the :kconfig:option:`CHIP_MALLOC_SYS_HEAP_WATERMARKS_SUPPORT` Kconfig option to manage watermark support.
+* Updated the factory data guide with an additional rotating ID information.
+* Fixed an IPC crash on nRF5340 when Zephyr's main thread takes a long time.
+* Fixed RAM and ROM reports.
+
+Thread
+------
+
+* Updated the Thread 1.3 compliant libraries.
+  The update contains a fix for network forming with legacy devices and key ID mode.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Added a workaround for known issue KRKNWK-16976 allowing to detect crashes of the nRF5340 network core running the 802.15.4 stack.
+
+Apple Find My
+-------------
+
+* Fixed two issues in the Apple Find My add-on for the |NCS|.
+  See the detailed description in the Find My release notes.
+
+Samples
+=======
+
+Matter samples
+--------------
+
+* :ref:`matter_lock_sample` sample:
+
+  * Fixed the feature map for software diagnostic cluster.
+    Previously, it was set incorrectly.
+  * Fixed the cluster revision for basic information cluster.
+    Previously, it was set incorrectly.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
+It also contains some |NCS| specific additions and commits cherry-picked from the upstream Zephyr repository including the following one:
+
+* Adjusted WEST_PYTHON to posix path to be consistent with other Python scripts that pass paths to the Zephyr CMake build system.
```

Comment:
There is one additional change merged to `sdk-zephyr` just now: https://github.com/nrfconnect/sdk-zephyr/commit/bb6a7db2c510c743374e9a8340ac065e6ae996c8
@ankuns @b-gent Shall it be reflected here?

Reply:
Possibly:
```
* Added a workaround against nRF5340 network core crashes which occurred after waking up from sleep state on WFI/WFE instructions.
```

Reply:
Alright, I will add that

Reply:
@b-gent We'd also need an entry here for https://github.com/nrfconnect/sdk-zephyr/pull/1236, as this will be merged. This change simply "fixes building with picolib enabled on Windows".

Reply:
will do